source/2019-04-12-the-ember-times-issue-94.md

```diff
@@ -169,6 +170,17 @@ Try out [Embroider](https://github.com/embroider-build/embroider) today!
 
 ---
 
+## [@action, {{on}} & {{fn}} blog post 🚀](https://www.pzuraq.com/ember-octane-update-action/)
+If you haven't been following the discussion on @action, {{on}}, and {{fn}}, this is the blog post for you!
+
+[@pzuraq](https://github.com/pzuraq) continues his wonderful series of blog posts with a new one that focuses on the evolution of @action, {{on}}, and {{fn}} in detail.
+
+The post covers some of the rationales for changing the use of {{action}} and how the use of the @action decorator will differ, including consideration of what the responsibilities of actions are and treatment of how binding works in this area of Ember. You also won't want to miss coverage of the {{on}} modifier that replaces event handling responsibilities of {{action}} and the {{fn}} partial application helper, both of which have RFC's that have moved into Final Comment Period.
```

Comment:
The meaning of "including consideration [...] this area of Ember" wasn't clear to me. I wonder if rewording this part or turning it into a new sentence might help?

(Can I ask what "this area" refers to? Is it `actions`?)

Reply:
Thanks for the review @ijlee2!  Yeah I think I meant it to refer to actions, but I suppose that really means binding of the context of `this` in templates.

Reply:
@ijlee2 I made a few tweaks. Does that read any more clearly to you?

Reply:
Yep, thanks!